mkdocs.yml

```diff
@@ -46,7 +46,6 @@ nav:
         - Electrical Overview: hw/electrical.md
         - Buttons and Light Ring: hw/face.md
         - Adapter Board: hw/adapter.md
-      - Hookup Examples: hw/hookup.md
```

Comment:
Is there any reason not to leave this file visible in the YAML? Might help to get people to keep it up to date, too.

Reply:
Well, I suppose it really comes down to views on navigation "cleanliness". 
While including it in the .yml would allow for easier visibility to it, the question becomes, is this for broken redirect links, or does it deserve it's own navigation page?

I tend to prefer a more cleaner navigation tree, but I can also see the two sides of it.

Reply:
Or "Hookup Guides?"